redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+                ).format(users=humanize_list([f"{u}" for u in users]))
+                return await self.handle_issues(ctx, message, issue)
+
+            if ctx.guild.id in self._server_mutes and self._server_mutes[ctx.guild.id]:
+                await self.config.guild(ctx.guild).muted_users.set(
+                    self._server_mutes[ctx.guild.id]
+                )
+            else:
+                await self.config.guild(ctx.guild).muted_users.clear()
+            await ctx.send(
+                _("{users} unmuted in this server.").format(
+                    users=humanize_list([f"{u}" for u in success_list])
+                )
+            )
+            if issue:
+                message = _(
+                    "{users} could not be unmuted in some channels. "
+                    "Would you like to see which channels and why?"
+                ).format(users=humanize_list([f"{u}" for u in users]))
+                await self.handle_issues(ctx, message, issue)
+
+    @checks.mod_or_permissions(manage_roles=True)
+    @commands.command(name="unmutechannel", aliases=["channelunmute"], usage="[users...] [reason]")
+    @commands.guild_only()
+    async def unmute_channel(
+        self,
+        ctx: commands.Context,
+        users: commands.Greedy[discord.Member],
+        *,
+        reason: Optional[str] = None,
+    ):
+        """Unmute a user in this channel.
+
+        `[users...]` is a space separated list of usernames, ID's, or mentions.
+        `[reason]` is the reason for the unmute.
+        """
+        if not users:
+            return await ctx.send_help()
+        if ctx.me in users:
+            return await ctx.send(_("You cannot unmute me."))
+        if ctx.author in users:
+            return await ctx.send(_("You cannot unmute yourself."))
+        if not await self._check_for_mute_role(ctx):
+            return
+        async with ctx.typing():
+            channel = ctx.channel
+            author = ctx.author
+            guild = ctx.guild
+            audit_reason = get_audit_reason(author, reason)
+            success_list = []
+            for user in users:
+                success = await self.channel_unmute_user(
+                    guild, channel, author, user, audit_reason
+                )
+
+                if success["success"]:
+                    success_list.append(user)
+                    try:
+                        await modlog.create_case(
+                            self.bot,
+                            guild,
+                            ctx.message.created_at.replace(tzinfo=timezone.utc),
+                            "cunmute",
+                            user,
+                            author,
+                            reason,
+                            until=None,
+                            channel=channel,
+                        )
+                    except RuntimeError as e:
+                        log.error(_("Error creating modlog case"), exc_info=e)
```

Comment:
Logging should not be translated.
```suggestion
                        log.error("Error creating modlog case", exc_info=e)
```